test/transcoding/SPV_INTEL_bfloat16_conversion/convert_bfloat16_generic.ll

```diff
@@ -0,0 +1,65 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv --spirv-ext=+SPV_INTEL_bfloat16_conversion
+; RUN: llvm-spirv %t.spv -o %t.spt --to-text
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.spv -o %t.rev.bc -r
+; RUN: llvm-dis %t.rev.bc -o %t.rev.ll
+; RUN: FileCheck < %t.rev.ll %s --check-prefix=CHECK-LLVM
+
+; RUN: not --crash llvm-spirv %t.bc 2>&1 | FileCheck %s --check-prefix=CHECK-ERROR
```

Comment:
If @svenvh doesn't mind I would suggest to adapt his change in lib/SPIRV/libSPIRV/SPIRVModule.cpp from https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1094/ to replace assertion with an error

Reply:
Thank you, applied Sven's change.